code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -375,7 +376,7 @@
 
 /datum/action/innate/ai/lockdown
 	name = "Lockdown"
-	desc = "Bolts, depowers, and electrifies every airlock, firelock, and blast door on the station. After 90 seconds, they will reset themselves."
+	desc = "Closes, bolts, and depowers every airlock, firelock, and blast door on the station. After 90 seconds, they will reset themselves."
```

Comment:
After 90 seconds, they will all be forced open for a short time before resetting actually, and that should probably be information that gets included